php.ini-production

```diff
@@ -1791,7 +1791,7 @@ ldap.max_links = -1
 
 ; A bitmask, where each bit enables or disables the appropriate OPcache
 ; passes
-;opcache.optimization_level=0xffffffff
+;opcache.optimization_level=0x7FFFFFFF
```

Comment:
Assuming these are defaults, shouldn't this be 0x7FFFBFFF per https://github.com/php/php-src/blob/c9034c3b3344287a1a636e43dbcb781bcfbd31af/ext/opcache/Optimizer/zend_optimizer.h#L47?

Reply:
Oh, maybe.. is there a drawback to using all passes?

Reply:
@nikic Please clarify

Reply:
By default one optimization pass is disabled, which propagates constants like `define('FOO', 42)`. The reason is that if the constant has been defined previously (in a different script), the second define will only throw a warning and keep the old value. As such, this optimization does not preserve behavior precisely and is not enabled by default.

Reply:
@nikic Ok, adjusted it, and also changed dev ini.
Should be good to merge now..